deploy/backplane/lpsre/15-lpsre-package-operator.SubjectPermission.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-lpsre-package-operator
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  # Some Package Operator APIs are cluster scoped,
+  # so access must be granted on the whole cluster.
+  - backplane-lpsre-package-operator-cluster
```

Comment:
Broad access to read and update resources in a customer namespace should not be granted without justification.  I don't see justification for access to this within customer namespaces.  I expect this request to be scoped to `(^kube$|^kube-.*|^openshift$|^openshift-.*|^default$|^redhat-.*)` namespaces.

Reply:
@jewzaam Sorry for the late response.
Pulled namespace-scoped resources into a new `backplane-lpsre-package-operator-project` role and added your suggested namespace filter :)
So now only cluster-scoped API are granted cluster-scoped.